EXCEPTION LIST
The path 'foo' was not found in the trash directory '<dir>/l3/.trash' despite being listed in the trash index. This can be fixed by manually deleting the entry from the index or deleting everything (i.e. sr e).

CallStack (from HasCallStack):
  getCallStack, called at src/SafeRm/Effects/MonadCallStack.hs:0:0 in <package>:SafeRm.Effects.MonadCallStack
  throwCallStack, called at src/SafeRm/Data/Index.hs:0:0 in <package>:SafeRm.Data.Index
  throwIfTrashNonExtant, called at src/SafeRm/Data/Index.hs:0:0 in <package>:SafeRm.Data.Index

LOGS LIST
[2020-05-31 12:00:00][functional.getIndex][Debug][src/SafeRm.hs] Index path: <dir>/l3/.trash/.index.csv
[2020-05-31 12:00:00][functional.getIndex.readIndex][Debug][src/SafeRm/Data/Index.hs] Index path: <dir>/l3/.trash/.index.csv
[2020-05-31 12:00:00][functional.getIndex.readIndex.readIndexWithFold][Debug][src/SafeRm/Data/Index.hs] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "foo"}, originalPath = MkPathI {unPathI = "<dir>/l3/.trash/foo"}, created = MkTimestamp {unTimestamp = 2022-09-28 02:58:33}}
[2020-05-31 12:00:00][functional][Error][src/SafeRm/Runner.hs] The path 'foo' was not found in the trash directory '<dir>/l3/.trash' despite being listed in the trash index. This can be fixed by manually deleting the entry from the index or deleting everything (i.e. sr e).

CallStack (from HasCallStack):
  getCallStack, called at src/SafeRm/Effects/MonadCallStack.hs:0:0 in <package>:SafeRm.Effects.MonadCallStack
  throwCallStack, called at src/SafeRm/Data/Index.hs:0:0 in <package>:SafeRm.Data.Index
  throwIfTrashNonExtant, called at src/SafeRm/Data/Index.hs:0:0 in <package>:SafeRm.Data.Index
